### PR TITLE
refine disabling gpu & fix render-process-gone logging

### PR DESCRIPTION
**Pull request by GrzegorzKozub**
Relates to #1369. Example of the log message:

```
08:03:09.099 › render-process-gone {"reason":"killed","exitCode":9}
```

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -203,9 +204,10 @@ async function playNotificationSound(_event, options) {
 	console.debug('No notification sound played', player, options);
 }
 
-function onRenderProcessGone(event, details) {
-	console.error(`render-process-gone with reason: '${details.reason}'.`);
-	console.error(`Event '${event}' and details '${details}'`);
+function onRenderProcessGone(event, webContents, details) {
+	// https://www.electronjs.org/docs/latest/api/app#event-render-process-gone
```
Comment (IsmaelMartinez) [inline app/index.js]: I don't think we need this commented lines. Is ok to remove them

**Comment by IsmaelMartinez** [comment]
Thanks @GrzegorzKozub for contributing. Would you be able to increase the version number and update the appdata files? There is a bit more info in https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/CONTRIBUTING.md#release-process 

Thanks again!

File: app/index.js (hunk 1 above)
Comment (GrzegorzKozub) [reply app/index.js]: Sure

**Comment by GrzegorzKozub** [comment]
> Thanks @GrzegorzKozub for contributing. Would you be able to increase the version number and update the appdata files? There is a bit more info in https://github.com/IsmaelMartinez/teams-for-linux/blob/develop/CONTRIBUTING.md#release-process
> 
> Thanks again!

Done

**Comment by IsmaelMartinez** [comment]
Thanks for the MR I will merge it now and hopefully release it later. Takes long enough (5 minutes) for me to forget I have trigger it 😄

**Comment by IsmaelMartinez** [comment]
yep, I did forget but is now out! Thanks again for contributing https://github.com/IsmaelMartinez/teams-for-linux/releases/tag/v1.9.6

**Pull request merged**